server/odc-service/src/main/java/com/oceanbase/odc/service/resultset/ResultSetExportTaskParameter.java

```diff
@@ -49,6 +49,7 @@ public class ResultSetExportTaskParameter implements Serializable, TaskParameter
     private List<MaskingAlgorithm> rowDataMaskingAlgorithms;
     @JsonIgnore
     private String database;
+    private Integer executionTimeoutSeconds = 48 * 60 * 60;
```

Comment:
48h seems too long, we cannot accept that init a task needing 48 hours.

Reply:
Maybe we cannot avoid this. If user requests to export a query with `/*+ timeout */` hint, it would be executed for a long duration anyway.
And this timeout param does not take effect during task initialization, but rather during task execution. So it should not have a serious impact.

Reply:
I got you!